_episodes/14-modules.md

```diff
@@ -173,7 +173,21 @@ Let's examine the output of `module avail` more closely.
 ```
 {: .output}
 
-{% include /snippets/14/gcc_example.snip %}
+To be more specific, you can specify the particular software you want.
+e.g.
+```
+{{ site.host_prompt }} module avail stata
+```
+{: .bash}
+```
+------------------------------------- /shared/ucl/apps/modulefiles/applications --------------------------------------
+stata/14 stata/15
+```
+{: .output}
+
+```
+{% include /snippets/14/matlab_example.snip %}
+```
```

Comment:
All these changes break the generalisation of the lesson, this should be stored into a snippet. The snippet `gcc_example.snip` could have all that and the content from matlab. So the same file it still works for the lesson upstream.

Reply:
Are we talking specifically of lines 178 - 186?